internal/lockclient/simple_client_test.go

```diff
@@ -36,32 +36,32 @@ func TestAcquireandRelease(t *testing.T) {
 	t.Run("acquire test release test", func(t *testing.T) {
 		size := 5
 		cache := cache.NewLRUCache(size)
-		sc := NewSimpleClient(*scfg, *cache)
+		sc := NewSimpleClient(scfg, cache)
 
-		d := lockservice.NewSimpleDescriptor("test", "owner")
+		d := lockservice.NewLockDescriptor("test", "owner")
 
-		got := sc.Acquire(d)
+		got := sc.acquire(d)
```

Comment:
Shouldn't `Acquire()` be called and not `acquire()`?

Reply:
Yep. This was when pounce existed.